pre-commit.sh

```diff
@@ -13,10 +13,10 @@ if [[ ! -z $1 ]]
 then
   bundle --quiet
   mkdir -p testreport
-  SHOW_BROWSER=${SHOW_BROWSER:-false} TEST_ENV=${TEST_ENV:-local} bundle exec cucumber $* --strict --tags 'not @ignore'
+  SHOW_BROWSER=${SHOW_BROWSER:-false} TEST_ENV=${TEST_ENV:-local} bundle exec cucumber $* --strict --tags 'not (@Eidas or @staging-only)'
```

Comment:
Should we keep the ability to ignore tests marked `@ignore`, too?

Reply:
Good quesion @instantiator . I should have mentioned in the PR that the only tests tagged @ignore are eidas tests, so we want to tag them as eidas explicity